Makefile

```diff
@@ -204,3 +204,29 @@ migrate: migrate-lms migrate-cms
 # Part of https://github.com/openedx/wg-developer-experience/issues/136
 ubuntu-requirements: ## Install ubuntu 22.04 system packages needed for `pip install` to work on ubuntu.
 	sudo apt install libmysqlclient-dev libxmlsec1-dev
+
+eslint:	## check javascript for quality issues
```

Comment:
The standard interface from our other frontend repos is to add the linting & testing scripts to package.json, such that users execute them with `npm run ...`.

Instead of adding Makefile targets for frontend scripts, could you add them to [package.json](https://github.com/openedx/edx-platform/blob/master/package.json#L5)? To be consistent with other repos, use:

* `npm run test` (for test-js)
* `npm run coverage` (for coverage-js)
* `npm run lint` (for eslint, stylelint, and xsslint)